lib/application.js

```diff
@@ -64,13 +64,19 @@ Application.prototype.startChromeDriver = function () {
     self.chromeDriver = new ChromeDriver(self.host, self.port)
     self.chromeDriver.start()
 
-    var waitUntilRunning = function () {
+    var tries = 0
```

Comment:
This should be configurable via an option like `startTimeout` passed into the `Application` so it can be tuned on CI in case 10 seconds isn't long enough.